src/components/forms/checkbox/checkbox.tsx

```diff
@@ -1,61 +1,93 @@
 import React, { useState } from 'react';
-import styled from 'styled-components';
+import { useController, UseControllerProps } from 'react-hook-form';
+import styled, { css } from 'styled-components';
+import { theme } from '../../layout/theme';
 import { TextStyles } from '../../typography';
 import { Icon } from '../../ui/icon/icon';
+import { StyledErrorMessage } from '../text-input/text-input';
 
-export const Checkbox = (props) => {
-  const [checked, setChecked] = useState(true);
+const CheckboxLabel = styled.label`
+  display: inline-flex;
+  align-items: center;
+`;
+
+const CheckboxLabelText = styled.span`
+  ${TextStyles.textR}
+
+  color: #202840;
+  margin-left: 12px;
+`;
+
+const StyledCheckbox = styled.div<{ checked: boolean; hasError: boolean }>`
```

Comment:
+ The checkbox is missed when I tab through the form
+ There is no focus highlight. We have a yellow border when interacting with the keyboard. We need that to signal where the focus sits right now
+ Is it feasible to deliver only this checkbox first? That would greatly simplify the review process as you could do the same for the other components too.